# pos/direction of objects(Monocular Vision)

Header header

# pos/direction of targets detected(num --- number of targets detected)

uint8 target_num 
uint32[] target_img_x
uint32[] target_img_y
int32[] target_dir_x
int32[] target_dir_y
uint32[] target_id
uint32[] target_color # 1 is red, 2 is green and 0 means there are some errors in the program.
int32[] target_area
int32[] flag_xinrencike
